### PR TITLE
Make Package.swift file consistent

### PR DESCRIPTION
**Pull request by aciidb0mb3r**

### CODE REVIEW

**Comment by normanmaurer** [comment]
@aciidb0mb3r sorry but I am not convinced this change is something we should take just for the sake of formatting. Can you explain me if there are any "fixes" in there that are not related to formatting ?

File: Package.swift [hunk 1]
```diff
@@ -15,57 +16,9 @@
...
-]
-
 let package = Package(
     name: "swift-nio",
     products: [
-        .executable(name: "NIOEchoServer", targets: ["NIOEchoServer"]),
-        .executable(name: "NIOEchoClient", targets: ["NIOEchoClient"]),
-        .executable(name: "NIOChatServer", targets: ["NIOChatServer"]),
-        .executable(name: "NIOChatClient", targets: ["NIOChatClient"]),
-        .executable(name: "NIOHTTP1Server", targets: ["NIOHTTP1Server"]),
```
[the remark this answers is not in the document]
Comment (normanmaurer) [reply Package.swift]: Because you a user want to run these from the command line ?

[the remark this answers is not in the document]
Comment (aciidb0mb3r) [reply Package.swift]: The products section is for "exporting" the products to your clients. The executable targets will always be built, regardless of an entry in the products section. Usually, libraries don't export executable products which are sample code. One reason to export executables is if they provide some helper tool that will be needed by the clients. 

Anyway, I thought NIO shouldn't be exporting the sample code but wasn't sure if there is a valid reason for doing so.

**Comment by aciidb0mb3r** [comment]
The package manager is trying to push for a declarative format for the `Package.swift` manifest file. IMO it is better to format NIO's manifest file this way so its consistent with how other community packages write the manifest. We want name to be seen first, then products, then its dependencies, and then the targets as they're not really that interesting for user of a package user.

But no, there is no actual "fix" that this PR makes. Feel free to close it!

**Comment by Lukasa** [comment]
I think we want to leave this as-is for now. Thanks @aciidb0mb3r!

**Pull request closed without merging**